dlt/dlt.py

```diff
@@ -948,7 +963,10 @@ def connect(self, timeout=None):
                         # dlt_client_connect() behavior
                         connected = dltlib.dlt_receiver_init(ctypes.byref(self.receiver),
                                                              self.sock,
-                                                             DLT_CLIENT_RCVBUFSIZE)
+                                                             receiver_type,
+                                                             DLT_RECEIVE_BUFSIZE)
```

Comment:
Does the function interface change between 2.18.5 and 2.18.6?
If yes, please call it by version. (The example is as the below.)